src/sentry/tasks/post_process.py

```diff
@@ -176,59 +178,64 @@ def post_process_group(event, is_new, is_regression, is_sample, is_new_group_env
         # we process snoozes before rules as it might create a regression
         has_reappeared = process_snoozes(event.group)
```

Comment:
Shouldn't this also be contained within the `event.group_id` conditional? It doesn't seem like `process_snoozes` expects a nullable group.

Reply:
thanks for noticing. Yes it does, this is rebase mistake. Will fix